code/modules/antagonists/clockcult/clock_helpers/slab_abilities.dm

```diff
@@ -179,6 +179,7 @@
 	. = ..()
 
 /obj/item/projectile/kindle/on_hit(atom/target, blocked = FALSE)
+	var/narsiandog_punish = 8
```

Comment:
no reason to do this, unnecessary variables.
also down below do something like C.confused += CLAMP(16 - C.confused, 0, 8) to prevent infinite stacking.